### PR TITLE
Update cStor user guides

### PR DESCRIPTION
**Pull request by anupriya0703**
The following subsections have been updated:
- [x] Block device tagging
- [x] Scaling up cStor pools
- [x] Deploying a sample application
- [x]  Performance Tunings in cStor Volumes
                - Replica Affinity to create a volume replica on specific pool
                - Volume Target Pod Affinity
                - Volume Tunable
                - Memory and CPU Resources QoS
                - Toleration for target pod to ensure scheduling of target pods on tainted nodes
                - Priority class for volume target deployment 
                
- [x] Cleaning up a cStor setup

Signed-off-by: anupriya0703 <anupriya.gupta@mayadata.io>

### CODE REVIEW

File: docs/ugcstor-csi.md [hunk 1]
```diff
@@ -5,50 +5,60 @@ sidebar_label: cStor
...
-
-   :::note 
-   The recommended approach to provision cStor Pools is to use CStorPoolCluster(CSPC), the detailed steps have been provided in this document. However, OpenEBS also supports provisioning of cStor Pools using StoragePoolClaim (SPC). For detailed instructions, refer to the <a href="https://docs.openebs.io/v260/docs/next/ugcstor.html" target="_blank">cStor User guide(SPC)</a>.<br>
-   :::
-
+ 
+This user guide section provides the operations needed to be performed by the User and the Admin for configuring cStor related tasks.
+ 
+  :::note
+  The recommended approach to provision cStor Pools is to use CStorPoolCluster(CSPC), the detailed steps have been provided in this document. However, OpenEBS also supports provisioning of cStor Pools using StoragePoolClaim (SPC). For detailed instructions, refer to the <a href="https://docs.openebs.io/v260/docs/next/ugcstor.html" target="_blank">cStor User guide(SPC)</a>.<br>
```
Comment (kmova) [inline docs/ugcstor-csi.md:17]:
```suggestion
  The recommended approach to provision cStor Pools is to use CStorPoolCluster(CSPC), the detailed steps have been provided in this document. However, OpenEBS also supports provisioning of cStor Pools using StoragePoolClaim (SPC). For detailed instructions, refer to the <a href="/docs/next/ugcstor.html" target="_blank">cStor User guide(SPC)</a>.<br>
```

File: docs/ugcstor-csi.md [hunk 2]
```diff
@@ -5,50 +5,60 @@ sidebar_label: cStor
...
+  :::
+
```
Comment (kmova) [inline docs/ugcstor-csi.md:19]: Remove the user and admin sections. Everything can just flow in as a list of operations. 

Move the snapshot and clone- after the `Scaling up cStor pools`

File: docs/ugcstor-csi.md (hunk 1 above)
Comment (anupriya0703) [reply docs/ugcstor-csi.md:17]: @kmova, We need to refer to older versions as when this doc will be pushed the SPC content will be entirely replaced with CSPC..Hence we have pointed to  OpenEBS doc version 2.6.0.

Comment (kmova) [reply docs/ugcstor-csi.md:17]: got it. I think we should move the older doc under the "deprecated" section.

File: docs/releases-0x.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 id: releases-0x
 title: OpenEBS 0.x Deprecated Releases
-sidebar_label: Releases 0.x
+sidebar_label: Summary of changes for releases 0.x
```
Comment (kmova) [inline docs/releases-0x.md:4]: n_: `0.x Release Notes`

File: docs/releases-1x.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
 id: releases-1x
 title: OpenEBS 1.x Deprecated Releases
-sidebar_label: Releases 1.x
+sidebar_label: Summary of changes for releases 1.x
```
Comment (kmova) [inline docs/releases-1x.md:4]: n_: `1.x Release Notes`

**Pull request merged**